tests/driver_at86rf2xx/ng_at86rf2xx_params.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   tests_at86rf2xx
+ * @brief     generic at86rf231 pin config
+ * @{
+ * @file
+ * @author Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef NG_AT86RF2XX_PARAMS_H
+#define NG_AT86RF2XX_PARAMS_H
+
+#warning Using default AT86RF2XX parameters.
+
+/**
+ * @name AT86RF231 configuration
+ */
+static const  at86rf2xx_params_t at86rf2xx_params[] =
+    {
+        {
+            .spi = SPI_0,
+            .spi_speed = SPI_SPEED_1MHZ,
+            .cs_pin = GPIO_0,
+            .int_pin = GPIO_1,
+            .sleep_pin = GPIO_2,
+            .reset_pin = GPIO_3,
```

Comment:
You don't break it directly, but you change the functionality slightly: If I want to test the radio with say the stm32f4discovery, and I connect it to SPI_1, GPIO_9-12. Then I would now need to change the header file. In the old test, all I had to do is define 5 environment variables on my shell and didn't have to change one line of code/Makefile to run it...